public/about.html

```diff
@@ -33,4 +33,11 @@
       </div>
     </div>
   </body>
+  <footer>
+    <div class="container giphy">
```

Comment:
That is to say, in the stylesheet the `.container` selector is only given properties when it has a parent `.about` selector, and I don't see a `.giphy` selector in the stylesheet. So it seemed safe to remove to me. 🐙